### ISSUE TITLE
taint: add @psalm-taint-escape html for e() helper

### ISSUE THREAD

**Issue by alies-dev** [psalm/laravel-psalm-plugin#532]
## Problem

The `e()` helper function is Laravel's primary HTML sanitizer, used by Blade `{{ }}` output. It wraps `htmlspecialchars($value, ENT_QUOTES | ENT_SUBSTITUTE, 'UTF-8')`.

Without a `@psalm-taint-escape html` annotation, every `{{ $userInput }}` in Blade templates triggers a false-positive taint error, since Psalm cannot see that `e()` neutralizes HTML taint.

## Current state

The common helpers stub (`stubs/common/Support/helpers.stubphp`) has a comment: `// e: nothing to stub (taint analysis only)` but no actual taint annotation exists anywhere.

## Proposed fix

Create a taint stub (either in `stubs/taintAnalysis/` or add to the existing helpers stub) with:

```php
/**
 * @psalm-taint-escape html
 * @psalm-taint-escape has_quotes
 */
function e($value, $doubleEncode = true): string {}
```

## Methods

- `e($value, $doubleEncode = true): string` — `Illuminate\Support\helpers.php`

## Impact

**Critical** — eliminates the largest source of false positives in taint analysis for any Laravel project using Blade templates.
